File: services/src/main/java/com/senacor/code/fullstack/chat/ChatApplication.java
package com.senacor.code.fullstack.chat;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.web.servlet.FilterRegistrationBean;
import org.springframework.context.annotation.Bean;

@SpringBootApplication
public class ChatApplication {

	public static void main(String[] args) {
		SpringApplication.run(ChatApplication.class, args);
	}

	@Bean
	public FilterRegistrationBean commonsRequestLoggingFilterRegistration(RequestLoggingFilter filter) {
		return new FilterRegistrationBean(filter);
	}
}

File: services/src/main/kotlin/com/senacor/code/fullstack/chat/ChatApplication.kt
package com.senacor.code.fullstack.chat

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.boot.web.servlet.FilterRegistrationBean
import org.springframework.context.annotation.Bean

@SpringBootApplication
open class ChatApplication {

    @Bean
    open fun commonsRequestLoggingFilterRegistration(filter: RequestLoggingFilter): FilterRegistrationBean<*> =
        FilterRegistrationBean(filter)
}


fun main(args: Array<String>) {
    runApplication<ChatApplication>(*args)
}
